components/director/internal/repo/delete.go

```diff
@@ -78,16 +78,18 @@ func (g *universalDeleter) unsafeDelete(ctx context.Context, conditions Conditio
 		stmtBuilder.WriteString(" WHERE")
 	}
 
+	log.Debug("Add conditions to the query")
```

Comment:
Probably add conditions to the log otherwise it does not provide much value. Just tracing information.
Valid for other methods also.